src/ts/cmd/endpoint-registry/package.json

```diff
@@ -4,13 +4,14 @@
   "description": "",
   "main": "src/index.js",
   "scripts": {
-    "test": "mocha --require ts-node/register --exit test/unit/**/*.spec.ts",
+    "dev": "tsc-watch --onSuccess \"node ./dist/server.js\"",
+    "test": "npm run test-integration-only",
```

Comment:
Are unit tests omitted on purpose here? Are they not there yet?

Reply:
"test-unit-only" has been added below. There are no unit tests at the moment, so does not make sense to include them in the default.

Reply:
Still, the integration almost completely fail. See https://travis-ci.com/github/SAP/i40-aas/jobs/320927566#L1068

Reply:
Yes, this is because we as a team decided that "done criteria" do not apply for this backlog item.

Reply:
Was just checking that this behavior is intended.

Reply:
Tests for the fix part of the Registry API are now in place